components/list/list-item-mixin.js

```diff
@@ -265,9 +273,9 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				margin: 0 -12px;
 			}
 			.d2l-list-item-content-extend-separators [slot="outside-control-container"] {
-				border-left: none;
-				border-radius: 0;
-				border-right: none;
+				border-left: none !important;
+				border-radius: 0 !important;
+				border-right: none !important;
```

Comment:
The `!important` was used because otherwise the `border` from the new css rules below take priority.